### ISSUE TITLE
[request] add option to delete sounds

### ISSUE THREAD

**Issue by paul-schaaf** [chaosparrot/parrot.py#41]
option in main menu to delete sounds e.g. for sounds that did not work well (or more generally an option to manage sounds. could also offer rename option)

**Comment by AndrewDant** [comment]
For what it's worth, you can accomplish this yourself by going to data/recordings and deleting or renaming the folder for a given recording.

**Comment by paul-schaaf** [comment]
yes. the reason this would still be nice to have is because the user of parrot may not and should not need to know the internals of parrot. maybe parrot keeps some configuration for the recordings somewhere that should also be deleted (I don't think it does but again the user shouldn't be required to know this)
